app/models/graphql/analyzer.rb

```diff
@@ -0,0 +1,22 @@
+module Graphql
+  class Analyzer
+    def initialize(graphql_schema)
+      @graphql_schema = graphql_schema
+    end
+
+    def execute(*args)
+      instrumenter = Instrumentation.new
+      result = schema(instrumenter).execute(*args)
+      result['instrumentation'] = instrumenter
```

Comment:
This should be `extensions` as per the [GraphQL spec](http://facebook.github.io/graphql/October2016/#sec-Response-Format)